dsnp-wallet/dsnp-wallet/Source/ViewControllers/SettingsViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SettingsViewController.swift
+//  DSNP-Wallet
+//
+//  Created by Ryan Sheh on 5/31/22.
+//
+
+import Foundation
+import UIKit
+import DSNPWallet
+
+class SettingsViewController: UIViewController {
+    override func viewDidLoad() {
+        setViews()
+    }
+    
+    @objc func tappedLogOut(selector: UIButton?) {
+        do {
```

Comment:
1. We try to delete keys, then we delete pin, then we present confirmation alert. We should present alert first, then only act on confirmation.
2. If we fail to delete keys, we should not delete pin.
3. We have a central place to clear pin. Does it make sense to have a central place that will clear both keys and pin, so that we don't have to rely on caller to know to delete both?